### PR TITLE
Tech Animation

### PR DESCRIPTION
**Pull request by moralesbang**
## Summary

- Added animation to tech ocomponent

## Screenshots

Jul-17-2019 16-59-14

## Trello Card

https://trello.com/c/QSmfE2CS

### CODE REVIEW

File: maquetado/scss/_variables.scss [hunk 1]
```diff
@@ -4,3 +4,6 @@ $font-family: 'Roboto', sans-serif;
 $text-decoration-height: 90px;
 $text-decoration-width: 70px;
 $text-big: 24px;
+$text-medium: 16px;
+$tech-logo-size: 100px;
+$tech-decoration-offset: 11px;
```
Comment (gabrielizalo) [inline maquetado/scss/_variables.scss]: Can `11px` be `10px`?

Comment (moralesbang) [reply maquetado/scss/_variables.scss]: That can, but for be accurate I should consider the `border-width` of `tech::before`.

Comment (gabrielizalo) [reply maquetado/scss/_variables.scss]: Ok, but in next opportunities prefer measurements in 5 multiples.

File: maquetado/scss/_tech.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    left: -$tech-decoration-offset;
+    position: absolute;
+    transition: transform 0.5s ease;
+    top: -$tech-decoration-offset;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover::before {
+    $translation-final: ($tech-decoration-offset * 2) - 1;
```
Comment (henryzarza) [inline maquetado/scss/_tech.scss]: I love it 👏

**Pull request merged**